src/components/RegisterForm.vue

```diff
@@ -1,71 +1,109 @@
+                    v-model="username"
+                  />
+                </div>
               </div>
-            </div>
-            <div class="form-group row">
-              <label for="email" class="col-sm-12 col-md-4 col-form-label">
-                Email
-              </label>
+              <div class="form-group row">
+                <label for="email" class="col-sm-12 col-md-4 col-form-label">Email</label>
 
-              <div class="col-sm-12 col-md-8">
-                <input
-                  name="email"
-                  type="text"
-                  class="form-control"
-                  id="email"
-                  h-100
-                  placeholder="Email"
-                />
+                <div class="col-sm-12 col-md-8">
+                  <input
+                    name="email"
+                    type="text"
+                    class="form-control"
+                    id="email"
+                    h-100
+                    placeholder="email"
+                    v-model="email"
+                  />
+                </div>
               </div>
-            </div>
-            <div class="form-group row">
-              <label for="password" class="col-sm-12 col-md-4 col-form-label">
-                Password
-              </label>
-              <div class="col-sm-12 col-md-8">
-                <input
-                  name="password"
-                  type="password"
-                  class="form-control"
-                  id="password"
-                  placeholder="Password"
-                />
+              <div class="form-group row">
+                <label for="password" class="col-sm-12 col-md-4 col-form-label">Password</label>
+                <div class="col-sm-12 col-md-8">
+                  <input
+                    name="password"
+                    type="password"
+                    class="form-control"
+                    id="password"
+                    placeholder="password"
+                    v-model="password"
+                  />
+                </div>
               </div>
-            </div>
-            <div class="form-group row">
-              <div class="col-sm-12">
-                <button type="submit" class="btn btn-outline-success w-25">
-                  Register
-                </button>
+              <div class="form-group row">
+                <div class="col-sm-12">
+                  <button
+                    v-on:click="signUp"
+                    type="submit"
+                    class="btn btn-outline-success w-25"
+                  >Register</button>
+                </div>
               </div>
-            </div>
-          </form>
+            </form>
+          </div>
         </div>
       </div>
-    </div>
+    </form>
   </div>
 </template>
 
 <script>
+import * as firebase from "firebase/app";
+import "firebase/auth";
 export default {
-  name: "RegisterForm"
+  name: "signup",
```

Comment:
Tutaj to samo, nie zmieniajmy prosze nazw komponentow na inne niz nazwy plikow, zachowajmy konwencje